python-web-scraping-practical-introduction/beautifulsoup.py

```diff
@@ -0,0 +1,12 @@
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
```

Comment:
Applying `isort` reshuffles these and adds a space:

```suggestion
from urllib.request import urlopen

from bs4 import BeautifulSoup
```